joomla/joomla-cms
# [4.0] Template Tree list improvements

### Summary of Changes

- Rewritten all the jQuery in vanilla JS and moved the JS/CSS to separate files.
- Also fixed up some of the Bootstrap styling.

### Testing Instructions

- apply PR
- Go to **Extensions** >> **Templates** >> **Templates**
- Ensure the tree list views are still fully functional and there are no console log errors

@dgt41 - JS ok?
